org.eclipse.xtext.ui.shared/plugin.properties

```diff
@@ -4,6 +4,8 @@ XtextBuilder.name=Xtext Project Builder
 XtextNature.name=Xtext Project Nature
 XtextNature.description=Adds an icon decoration to projects with Xtext enabled.
 XtextNatureAddAction.label=Convert to Xtext Project
+XtextNatureAddCommand.label=Convert to Xtext Project
```

Comment:
I know there is a difference between the deprecated action and the new command, but shall we just reuse the message? It's just a "wrong" key name. What do you think?

Reply:
i think i even can remove the old key

Reply:
updated